src/ui/components/Events/Events.tsx

```diff
@@ -9,9 +9,24 @@ const { getExecutionPoint } = require("devtools/client/debugger/src/reducers/pau
 
 import Event from "./Event";
 import { trackEvent } from "ui/utils/telemetry";
+import classNames from "classnames";
 
-function CurrentTimeLine() {
-  return <div className="bg-secondaryAccent w-full m-0" style={{ height: "3px" }} />;
+function CurrentTimeLine({
+  currentEventIndex,
+  index,
+}: {
+  currentEventIndex: number;
+  index: number;
+}) {
+  return (
+    <div
+      className={classNames("w-full m-0", {
```

Comment:
Do you need the `w-full` here? I think the div should be full-width by default, as it's a block-level element.